apps/backend/src/mcp/tools/metrics_collection.py

```
@@ -126,7 +126,7 @@ async def get_system_info(device: str, timeout: int = 60) -> Dict[str, Any]:
         # Prepare response
         response = {
             "device": device,
-            "timestamp": datetime.now(datetime.UTC).isoformat(),
+            "timestamp": datetime.now(timezone.utc).isoformat(),
```

Ruff (0.12.2)

Undefined name `timezone`

(F821)